Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -15,10 +15,9 @@ import SwiftUI
 public struct MessagesView: View {
     private static let bottomSpacerIdentifier = "Bottom Spacer"
     
-    
     @Binding var chat: [Chat]
     @Binding var bottomPadding: CGFloat
```

Comment:
In line with the other property, you can also make these private:
```suggestion
    @Binding private var chat: [Chat]
    @Binding private var bottomPadding: CGFloat
```